Source/Parallelization/WarpXSumGuardCells.H

```diff
@@ -66,7 +68,7 @@ WarpXSumGuardCells(amrex::MultiFab& dst, amrex::MultiFab& src,
     else  // Update only the valid cells
         n_updated_guards = amrex::IntVect::TheZeroVector();
 
-    src.SumBoundary(0, ncomp, n_updated_guards, period);
+    src.SumBoundary(0, ncomp, src_ngrow, n_updated_guards, period);
     amrex::Copy( dst, src, 0, icomp, ncomp, n_updated_guards );
```

Comment:
```suggestion
    dst.setVal(0., icomp, ncomp, n_updated_guards);
    dst.ParallelAdd(src, 0, icomp, ncomp, src_ngrow, n_updated_guards, period); 
```

Reply:
Thank you, Weiqun! Done in https://github.com/ECP-WarpX/WarpX/pull/2323/commits/76935032558f5f6115c56e5ebdc4e8f01499008f, together with the reversion of https://github.com/ECP-WarpX/WarpX/commit/4060bb13a6717b0602c9cd7eecd45c3420364941, due to the reasons explained in https://github.com/ECP-WarpX/WarpX/pull/2323#issuecomment-924448645.